neural_compressor/torch/quantization/quantize.py

```diff
@@ -81,5 +82,124 @@ def quantize(
                 run_fn=run_fn,
                 run_args=run_args,
                 example_inputs=example_inputs,
+                mode="quantize",
+            )
+    return q_model
+
+
+@log_quant_execution
```

Comment:
`log_quant_execution` prints something like `Quantization started`, `Quantization ended`. Suggest removing it first and making the `log_quant_execution` more general in a separate PR.

Reply:
removed